python/pyspark/ml/regression.py

```diff
@@ -755,11 +761,35 @@ class TreeEnsembleModels(JavaModel):
     """
 
     @property
+    @since("2.0.0")
+    def trees(self):
+        """Trees in this ensemble. Warning: These have null parent Estimators."""
+        return [DecisionTreeModel(m) for m in list(self._call_java("trees"))]
+
+    @property
+    @since("2.0.0")
+    def getNumTrees(self):
```

Comment:
@holdenk good catch to add this - can we add a doc-test for it too? Then I think this is good to go.